app/src/main/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityPresenter.kt

```diff
@@ -72,33 +94,89 @@ class AdministratorControlsActivityPresenter @Inject constructor(
       ) as AdministratorControlsFragment?
   }
 
+  /**
+   * Loads the profile list fragment as the [AdministratorControlsActivity] is
+   * started in multipane tablet mode.
+   */
   fun loadProfileList() {
     lastLoadedFragment = PROFILE_LIST_FRAGMENT
     getAdministratorControlsFragment()!!.setSelectedFragment(lastLoadedFragment)
+    setMultipaneBackButtonVisibility(View.GONE)
     activity.supportFragmentManager.beginTransaction().add(
       R.id.administrator_controls_fragment_multipane_placeholder,
       ProfileListFragment.newInstance(isMultipane)
     ).commitNow()
   }
 
+  /** Loads the [AppVersionFragment] in the multipane tablet mode. */
   fun loadAppVersion() {
     lastLoadedFragment = APP_VERSION_FRAGMENT
     getAdministratorControlsFragment()!!.setSelectedFragment(lastLoadedFragment)
+    setMultipaneBackButtonVisibility(View.GONE)
     activity.supportFragmentManager.beginTransaction().add(
       R.id.administrator_controls_fragment_multipane_placeholder,
       AppVersionFragment()
     ).commitNow()
   }
 
+  /** Loads the [ProfileEditFragment] when the user clicks on a profile in tablet multipane mode. */
+  fun loadProfileEdit(profileId: Int) {
+    lastLoadedFragment = PROFILE_EDIT_FRAGMENT
+    setMultipaneBackButtonVisibility(View.VISIBLE)
+    val fragment = ProfileEditFragment.newInstance(profileId, isMultipane)
+    activity.supportFragmentManager.beginTransaction().add(
+      R.id.administrator_controls_fragment_multipane_placeholder,
+      fragment
+    ).commitNow()
+  }
+
+  private fun setMultipaneBackButtonVisibility(visibility: Int) {
+    binding.administratorControlsMultipaneOptionsBackButton!!.visibility = visibility
+  }
+
+  /** Handles the back button according to the back stack of fragments. */
+  fun handleOnBackPressed() {
+    if (isMultipane) {
+      setBackButtonClickListener()
+    }
+  }
+
+  private fun setBackButtonClickListener() {
+    backButtonLogic()
+    binding.administratorControlsMultipaneOptionsBackButton!!.setOnClickListener {
+      backButtonLogic()
+    }
+  }
+
+  private fun backButtonLogic() {
+    val currentFragment =
+      activity.supportFragmentManager.findFragmentById(
+        R.id.administrator_controls_fragment_multipane_placeholder
+      )
+    if (currentFragment != null) {
+      when (currentFragment) {
+        is ProfileEditFragment -> {
```

Comment:
This can just be an if-check, right?

Reply:
when I'm going to add more fragments like ProfileRenameFragment and ProfileResetPinFragment then we are going to need when so I'm using it.

Reply:
Is that going to be happening in a follow-up PR, or is it an eventuality that could happen?

Also, what about the 'else' case--what should the presenter do in those cases? If shouldn't happen, consider adding a warning log line to help with debugging if it leads to a broken user experience.